lib/jekyll/assets/liquid/tag/proxied_asset.rb

```diff
@@ -109,8 +139,12 @@ def proxy_file
 
           private
           def cache_file
-            @_cached = File.file?(filename)
-            return @_cached if @_cached
+            glob = Dir.glob("#{filename.chomp(File.extname(filename))}.*")
```

Comment:
* If filename is just a filename: `Pathutil.new(".").glob(Pathutil.new(filename).sub_ext(".*"))`
* If filename is just a absolute: `filename.dirname.glob(filename.basename.sub_ext(".*"))`